server/schemas/user.schema.js

```diff
@@ -144,9 +143,7 @@ export const updateVendorSchema = Joi.object({
   vendor,
 });
 
-export const addDirectorSchema = Joi.object({
-  name: requiredString('Name'),
-  isSignatory: requiredBoolean('Signatory'),
-  signature: optionalString('Signature'),
-  phone: requiredString('Phone'),
+export const updateDirectorSchema = Joi.object({
+  ...directorSchema,
+  directorId: Joi.objectId().label('Director id').required(),
```

Comment:
I think it is preferable to use the exact name here for the id to make it easier to send from any part.
```suggestion
  _id: Joi.objectId().label('Director id').required(),
```